### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by roddicki** [tensorflow/tensorflow#55521]
Hi I've been trying to train tflite pose classifiers to work with your movenet pi example
https://github.com/tensorflow/examples/tree/master/lite/examples/pose_estimation/raspberry_pi

I've used the suggested tutorial and colab 
https://www.tensorflow.org/lite/tutorials/pose_classification
and carefully followed the example yoga pose dataset (with train and test examples) but the classification results seem almost random (2 classifiers out 4 successfully classify a pose about 50% of the time).

My dataset is here
https://github.com/roddicki/movenet-pi-dataset

The training in the colab works well with a confusion matrix with no errors. The model accuracy is also high (0.9977) and the increasing accuracy through the training epochs looks as expected. I wondered if anyone has used this colab with success?

Thanks!

**Comment by mihaimaruseac** [comment]
Hi. Please use [TF forum](https://discuss.tensorflow.org/) for these types of questions. GitHub is for bugs related to the code or feature requests.
